standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/CompactionInfo.java

```diff
@@ -116,6 +117,7 @@ public String toString() {
       "partName:" + partName + "," +
       "state:" + state + "," +
       "type:" + type + "," +
+      "queueTime:" + queueTime + "," +
```

Comment:
(Opportunity: "start" is missing from this method)

Reply:
Done